custom-namespaces.yaml

```
@@ -571,6 +571,13 @@ firefox_accounts:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.accounts_db.fxa_oauth_clients
+    monitor_db_counts:
```

I'd recommend not using "monitor" in the name, as that could be confused with the Monitor product (e.g. the `monitor_mozilla_accounts_funnels` view defined below is related to the Monitor product).
```suggestion
    database_counts:
```

---

I didn't love it, but was trying to mirror the source table name.  this wfm 👍

---

If you'd prefer to exactly mirror the the source table name and go with `monitoring_db_counts` that'd also be reasonable to me. It's `monitor` (without the `ing`) that I think would be confusing.

And in my suggested changes I proposed `database` rather than `db` because of the way Looker auto-converts view and explore names in LookML to title case the UI, and "Database Counts" seemed preferable to "Db Counts" to me.

---

I have no strong feelings.  It's under the accounts header, so, /shrug  Let's go with what we've got so we can land it